compiler/include/ForLoop.h

```diff
@@ -22,6 +22,11 @@
 
 #include "LoopStmt.h"
 
+// A ForLoop represents the for-statement language construct as described in
+// section 11.8 of the specification.  The buildForLoop() method transforms the
```

Comment:
While the section number is nice, I fear it'll become outdated pretty quickly and it's pretty easy to just search for ForLoop in the spec.